modis_lai/create_artifacts.jl

```diff
@@ -51,71 +58,82 @@ else
+    lai_data_regridded .= ifelse.(isnan.(lai_data_regridded), 0.0f0,
                                                              lai_data_regridded)
 
-# Since we regridded to half of the resolution of the original data, we need to 
-# average every 2 lat/lon points to get the lat/lon of the regridded data.
-out_lon = (orig_lon[1:2:end] .+ orig_lon[2:2:end]) ./ 2
-out_lat = (orig_lat[1:2:end] .+ orig_lat[2:2:end]) ./ 2
-
-# Write the regridded data to the output directory
-output_path = joinpath(OUTPUT_DIR, OUTPUT_FILE)
-ds          = NCDataset(output_path, "c")
-
-# Define the dimensions of the data -> 2D spatially varying data over 12 months
-defDim(ds, "lon", size(lai_data_regridded)[1])
-defDim(ds, "lat", size(lai_data_regridded)[2])
-defDim(ds, "time", 12)
-
-# Define the variables of the data - lat, lon, and the lai
-la      = defVar(ds, "lat", Float32, ("lat",))
-lo      = defVar(ds, "lon", Float32, ("lon",))
-month   = defVar(ds, "time", Int32, ("time",))
-lai_var = defVar(ds, "lai", Float32, ("lon", "lat", "time"))
-
-# Set the attributes of the variables.
-la.attrib["units"]             = "degrees_north"
-la.attrib["standard_name"]     = "latitude"
-lo.attrib["units"]             = "degrees_east"
-lo.attrib["standard_name"]     = "longitude"
-month.attrib["units"]          = "seconds since 1970-01-01"
-month.attrib["standard_name"]  = "time"
-month.attrib["calendar"]       = "proleptic_gregorian"
-lai_var.attrib["units"]         = "m^2 m^-2"
-lai_var.attrib["standard_name"] = "Leaf area index"
-
-# Write the data for each variable out to the nc file
-la[:]     = out_lat
-lo[:]     = out_lon
-# Note that even spacing on the timestamps is required for the periodicCalendar
-# boundary condition
-times     = [DateTime(2008, 1, 1) + Dates.Day(30 * (k - 1)) for k in 1:12]
-month[:]  = Int32.([Dates.datetime2unix(t) for t in times])
-lai_var[:, :, :] = lai_data_regridded
-close(ds)
-
-# Remove the initial downloaded artifact file - desired data is now stored in 
-# the CliMA artifact.
-Collector.clean_collections!(lai_collector)
+    # Since we regridded to half of the resolution of the original data, we need
+    # to average every 2 lat/lon points to get the lat/lon of the regridded
+    # data.
+    out_lon = (orig_lon[1:2:end] .+ orig_lon[2:2:end]) ./ 2
+    out_lat = (orig_lat[1:2:end] .+ orig_lat[2:2:end]) ./ 2
+
+    # Write the regridded data to the output directory
+    output_path = joinpath(OUTPUT_DIR, OUTPUT_FILE)
+    ds          = NCDataset(output_path, "c")
+
+    # Define the dimensions of the data -> 2D spatially varying data over 12
+    # months
+    defDim(ds, "lon", size(lai_data_regridded)[1])
+    defDim(ds, "lat", size(lai_data_regridded)[2])
+    defDim(ds, "time", 12)
+
+    # Define the variables of the data - lat, lon, and the lai
+    la      = defVar(ds, "lat", Float32, ("lat",))
+    lo      = defVar(ds, "lon", Float32, ("lon",))
+    month   = defVar(ds, "time", Int32, ("time",))
```

Comment:
Calling this the month is a little confusing to me, because it's attrib below is "time", "seconds since ref date"

Reply:
oh I see! the git diff was not showing me the loop over years

we might leave a comment (if it isnt there already, not in the git diff) that the data from gridding machine is in month of the year (1:1:12), and we need a DateTime